base/applications/rapps/unattended.cpp

```diff
@@ -253,10 +259,17 @@ ParseCmdAndExecute(LPWSTR lpCmdLine, BOOL bIsFirstLaunch, int nCmdShow)
             /* Activate window */
             ShowWindow(hWindow, SW_SHOWNORMAL);
             SetForegroundWindow(hWindow);
+            if (bAppwizMode)
+                PostMessage(hWindow, WM_COMMAND, ID_ACTIVATE_APPWIZ, 0);
             return FALSE;
         }
 
-        MainWindowLoop(&db, nCmdShow);
+        CMainWindow *wnd = new CMainWindow(&db, bAppwizMode);
```

Comment:
Don't need to manually allocate this anymore, can just create it on the stack:
```suggestion
        CMainWindow wnd(&db, bAppwizMode);
```